docs/USAGE-GUIDE.md

```diff
@@ -492,7 +492,8 @@ preference, with most preferred protocol first, and of length
 **protocol_count**.  When acting as an **S2N_CLIENT** the protocol list is
 included in the Client Hello message as the ALPN extension.  As an
 **S2N_SERVER**, the list is used to negotiate a mutual application protocol
-with the client.
+with the client. After the negotiation for the connection has completed, the 
```

Comment:
We're currently using the USAGE-GUIDE for external documentation. I think we should avoid references to how data is stored internally in opaque s2n objects. What do you think about this instead:

```
After the negotiation for the connection has completed, the
agreed upon protocol can be retrieved with [s2n_get_application_protocol](#s2n_get_application_protocol)
```

The markdown link to section will *hopefully* work when it is actually rendered

Reply:
thats fine. I actually missed that function when grepping for alpn usage. That’s much better.